### DISCUSSION TITLE
Missing vulnerability in package of an image

### DISCUSSION THREAD

**Discussion by savujevi** [DependencyTrack/dependency-track#1196] [Q&A]
Hello developers,
I have import an bom.json to dependency-track produced by syft and I am missing a vulnerability in the rpm:

| libX11        | CVE-2021-31535   | HIGH     | 1.6.7-3.el7_9

This CVE was detected in the image by trivy.

The details of the package produced by syft are:

purl: pkg:rpm/redhat/libX11@1.6.7-3.el7_9?arch=x86_64

All Scanners are turned on. Which of the scanners and what database is used for rpms ?

Thank you for your help.

Best regards.
   Sascha Vujevic

**Comment by stevespringett** [comment]
All components that have only a PURL defined will use OSS Index as the primary source of vulnerability intelligence. If you believe this is a false negative, please report `pkg:rpm/redhat/libX11@1.6.7-3.el7_9?arch=x86_64` to the OSS Index project. Once fixed, the component will show up as vulnerable in Dependency-Track. 

See https://ossindex.sonatype.org/doc/report-vulnerability

**Reply by savujevi** [reply]
Thank you for the fast reply.

Could you please tell me, which data is needed to use other indexes ?
Maybe it is documented.

Thanks and best regards.
   Sascha Vujevic
